doc/DESI_TARGET/targets.rst

```diff
@@ -39,65 +32,113 @@ FITS Header Units
-RA_IVAR               float32          Inverse variance of RA
-DEC                   float64          Declination [degrees]
-DEC_IVAR              float32          Inverse variance of DEC
-DECAM_FLUX            float32[6]       DECam flux from tractor input (ugrizY)
-DECAM_MW_TRANSMISSION float32[6]       Milky Way dust transmission ([0-1] ugrizY)
-WISE_FLUX             float32[4]       WISE flux (W1, W2, W3, W4)
-WISE_MW_TRANSMISSION  float32[4]       Milky Way transmission
-SHAPEEXP_R            float32          Half-light radius of exponential model (>0)
-SHAPEDEV_R            float32          Half-light radius of deVaucouleurs model (>0)
-TARGETID              int64            ID (unique to file and the whole survey)
-DESI_TARGET           int64            DESI (dark time program) target selection bitmask
-BGS_TARGET            int64            BGS (bright time program) target selection bitmask
-MWS_TARGET            int64            MWS (bright time program) target selection bitmask
-===================== ========== ===== ===================
+================== ========== =============== ==================================
+Name               Type       Units           Description
+================== ========== =============== ==================================
+RELEASE            int32                      label for column  1
+BRICKID            int32                      Integer brick ID
+BRICKNAME          char[8]                    String brick name
+BRICK_OBJID        int32                      ObjectID on that brick
+TYPE               char[4]                    PSF, SIMP, DEV, EXP
+RA                 float64    deg             Right Ascension      
+DEC                float64    deg             Declination
+RA_IVAR            float32    1/deg2          Inverse variance of RA     
+DEC_IVAR           float32    1/deg2          Inverse variance of dec     
+DCHISQ             float32[5]                 
+FLUX_G             float32    nanomaggies     g-band flux
+FLUX_R             float32    nanomaggies     r-band flux      
+FLUX_Z             float32    nanomaggies     z-band flux
+FLUX_W1            float32    nanomaggies     WISE W1-band flux
+FLUX_W2            float32    nanomaggies     WISE W2-band flux
+FLUX_W3            float32    nanomaggies     WISE W3-band flux
+FLUX_W4            float32    nanomaggies     WISE W4-band flux
+FLUX_IVAR_G        float32    1/nanomaggies2  inverse variance of FLUX_G
+FLUX_IVAR_R        float32    1/nanomaggies2  inverse variance of FLUX_R
+FLUX_IVAR_Z        float32    1/nanomaggies2  inverse variance of FLUX_Z
+FLUX_IVAR_W1       float32    1/nanomaggies2  inverse variance of FLUX_W1
+FLUX_IVAR_W2       float32    1/nanomaggies2  inverse variance of FLUX_W2
+FLUX_IVAR_W3       float32    1/nanomaggies2  inverse variance of FLUX_W3
+FLUX_IVAR_W4       float32    1/nanomaggies2  inverse variance of FLUX_W4
+MW_TRANSMISSION_G  float32                    Milky Way transmission in g
```

Comment:
Could describe all of the rows...I think it's fine to let people look at legacysurvey.org, though